### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by mchompalova**
# Description

## What

Add runbook for the service.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-1181

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [x] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: .github/runbooks.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # defaults to `master`
+  updateBranch: master
+
+  ### UPDATING BIZ-OPS REQUIRES EACH RUNBOOK TO BE TIED TO A VALID SYSTEM CODE
+  # Mappings between paths and system codes
+  # unless a custom mapping is specified here
+  # runbook.md tries to parse the system code from the runbook's filename (format: my-sys-code_runbook.md)
+  systemCodes:
+    # paths are relative to root, omitting ./ (case-insensitive)
+    content-collection-unfolder: runbooks/runbook.md
```
Comment (ekaneva) [inline .github/runbooks.yml]: Missing new line at the end of file.

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+* upp-neo4j-cluster
+* upp-kafka
```
Comment (ekaneva) [inline runbooks/runbook.md]: Different dependancies are listed in the README:

> Dependencies are:
> 
> relations-api
> content-collection-rw-neo4j
> document-store-api
> kafka

**Review by ekaneva — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# UPP - Content Collection Unfolder
+
+Content Collection Unfolder finds added/deleted collection members and lead articles through the relations-api.
```
Comment (ekaneva) [inline runbooks/runbook.md]:
```suggestion
Content Collection Unfolder finds added/deleted collection members and lead articles through the `relations-api`.
```

File: runbooks/runbook.md (hunk 2 above)
Comment (mchompalova) [reply runbooks/runbook.md]: Fixed.

**Review by ekaneva — APPROVED** [review]
Something that I notice just now is that we mix the unordered list items notation (`-` with `*`).

It is like so for many (if not for all) prior runbooks, but since it does not matter for the runbook validator, we can live with this havoc. :D

Maybe just for future runbooks, it will be a good idea to match their style. Per our [Markdown guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/style-guides/markdown-style-guide#use---for-unordered-lists), the `-`es are the preferred notation.

**Review by bboykov — APPROVED** [review]

**Pull request merged**